topics/firefox/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: seamonkey, waterfox, palemoon, icecat, iceweasel, torbrowser
```

Comment:
`torbrowser` seems more like a related topic than an alias. Would you move it to line 11?

Reply:
torbrowser is just a tweaked firefox esr, Firefox extensions can be installed to it and its extensions (which are in fact Firefox extensions) can be installed on old (<57, where their support was dropped) Firefox.